### PR TITLE
Swagger config setup for local development

### PR DESCRIPTION
**Pull request by prv-proton**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "react-table": "^7.8.0",
     "resolve": "^1.20.0",
     "semver": "^7.3.5",
+    "swagger-ui-react": "^5.10.3",
```
Comment (hamed-valiollahi) [inline frontend/package.json:39]: Should we limit Swagger UI and FastAPI docs to just the development environment?

**Comment by hamed-valiollahi** [comment]
Great work, Prashanth! We really needed Swagger UI. I appreciate you getting it running on my local setup.

I have a suggestion regarding the route. Perhaps we could consider using `/api/docs` instead of `/docs`. However, the final decision is yours.

File: frontend/src/layouts/PublicLayout.jsx
[the remark this answers is not in the document]
Comment (prv-proton) [reply frontend/src/layouts/PublicLayout.jsx:1]: Yes, you're right. It was changes from my local moved by mistake.  for ApiDocs component I'm not enclosing it within the layout, instead checking for authentication directly. Feel free to suggest for any other changes.

File: frontend/src/layouts/Layout.jsx
[the remark this answers is not in the document]
Comment (prv-proton) [reply frontend/src/layouts/Layout.jsx:1]: changed to AuthLayout

File: frontend/package.json (hunk 1 above)
Comment (prv-proton) [reply frontend/package.json]: done. thanks for the suggestion

**Comment by prv-proton** [comment]
> Great work, Prashanth! We really needed Swagger UI. I appreciate you getting it running on my local.
> 
> I have a suggestion regarding the route. Perhaps we could consider using `/api/docs` instead of `/docs`. However, the final decision is yours.
I can, but unlike the backend, in frontend we would need to create the /api page and the child page would then become /api/docs.

**Review by kevin-hashimoto — APPROVED** [review]

**Pull request merged**